### ISSUE TITLE
Backing up MySQL Database from master server to the backup server

### ISSUE THREAD

**Issue by ebonsi** [sequelpro/sequelpro#2055]
Hi,
Thanks for the great application. Just to be free from phpMyadmin crazy configuration is already a victory! I am on the Mac OS X Mavericks, 10.9.5. The Sequel Pro Version is 1.0.2 Build 4096. Ok! The issue:
I am working with the ecommerce Prestashop and I am having a problem in making an exact backup copy from the master to load in the backup server. It is just not coming up right. A lot of products pictures did not backup and oddly the ones that backup are not fitting with the right product. Do I have to drop the entire database (just the way phpMyadmin requires) to load a new one or when I am backing up and load with Sequel Pro it will overwrite the old one? Anyway, is not working very well. Maybe I am doing something wrong! 

In addition, I cannot repair the Tables. It basically gives this warning;
"Error while repairing selected items;"
MySQL said:
prestashop.ps_access    note    The storage engine for the table doesn't support repair
prestashop.ps_accessory note    The storage engine for the table doesn't support repair
...
and there is the same warning for the whole list of Tables.

PS: If you need any more information feel free to ask, Thanks!

**Comment by willpower232** [comment]
"The storage engine for the table doesn't support repair" could mean you are attempting to apply an operation for MyISAM tables to a completely different type of table.

Are you not using mysqldump to create a backup of the database?

**Comment by ebonsi** [comment]
I have to confess that I am a little frustrated with the backup process. 

First I tried to backup from Prestashop itself, drop the tables and so forth. When I load the new data base to the backup server it just not come right. The structure is ok but there are too many missing pictures from products and the original configuration of the master.

2. Then I tried to backup from Sequel Pro itself, same, same thing.

3. The only thing I have not used is phpMyadmin because it is a rack but their backup is more consistent that all the other I have tried.

No, I am not using mysqldump. It seems to me to be another hack. I am trying to avoid phpMyadmin but it seems that will have to try backup thru them again.

**Comment by ebonsi** [comment]
Ok, I'll wait until someone will develop a better way to backup and restore databases with all the info in like configurations and photos etc. In the mean time I just go ahead in the terminal and use;
cp -R [source] [destination] and then adjust two parameters on the configuration to connect to MySQL.

**Comment by ABMysql** [comment]
Dear friends, 
Hello ! 

www.abmysql.com is a free service to run automatic backup and restore your mysql database.

**Comment by ebonsi** [comment]
Hello, I want to update this post as I found an easy way to do it. I am using "Sequel Pro" to do the backups and it is working very well after their last update. There are another way to back using Prestashop itself. Now be aware that you are backing up only the MySQL Database. You still have to copy the entire store and bring over to your other server since none of the backups I tested backup all the previous the images e configuration.
